frameworks/Java/redkale/config.toml

```diff
@@ -14,36 +14,35 @@ platform = "Redkale"
 webserver = "Redkale"
 versus = "Redkale"
 
-[postgres]
-urls.db = "/db"
-urls.fortune = "/fortunes"
-urls.query = "/queries?q="
-urls.update = "/updates?q="
+[cache]
```

Comment:
This config doesn't match the one above. Hard to have a cache test without a database 😆

Reply:
added， thanks